tests/TranslatableTest.php

```diff
@@ -414,6 +415,18 @@ public function it_has_methods_that_return_always_a_translation(): void
         static::assertEquals('xyz', $vegetable->translateOrNew()->locale);
     }
 
+    /** @test */
+    public function it_has_methods_that_throw_an_exception_if_translation_does_not_exist(): void
```

Comment:
`it_throws_an_exception_if_translation_does_not_exist`. Using `it_has_methods` doesn't seems right to me 🤔 You're not testing if those methods are added to the model, you're testing that those methods works properly